core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/test_engine/add_service_framework_test.go

```diff
@@ -112,8 +112,8 @@ func (t *addServiceTestCase) Assert(interpretationResult starlark.Value, executi
 	serviceObj, ok := interpretationResult.(*kurtosis_types.Service)
 	require.True(t, ok, "interpretation result should be a dictionary")
 	require.NotNil(t, serviceObj)
-	expectedServiceObj := fmt.Sprintf(`Service(hostname = "{{kurtosis:%s.hostname}}", ip_address = "{{kurtosis:%s.ip_address}}", ports = {%q: PortSpec(number=%d, transport_protocol=%q, application_protocol=%q)})`, TestServiceName, TestServiceName, TestPrivatePortId, TestPrivatePortNumber, TestPrivatePortProtocolStr, TestPrivateApplicationProtocol)
-	require.Equal(t, expectedServiceObj, serviceObj.String())
+	expectedServiceObj := fmt.Sprintf(`Service\(hostname = "{{kurtosis:[0-9a-f]{32}:hostname.runtime_value}}", ip_address = "{{kurtosis:[0-9a-f]{32}:ip_address.runtime_value}}", ports = {%q: PortSpec\(number=%d, transport_protocol=%q, application_protocol=%q\)}\)`, TestPrivatePortId, TestPrivatePortNumber, TestPrivatePortProtocolStr, TestPrivateApplicationProtocol)
```

Comment:
It's not printing, this is the regex we interpolate. This regex represents the UUIDs we generate for runtime value ids, since we don't want to pin to a specific one